src/AppInstallerCLICore/AppInstallerCLICore.vcxproj

```diff
@@ -226,6 +226,11 @@
       <SubSystem Condition="'$(Configuration)|$(Platform)'=='Release|x64'">Windows</SubSystem>
     </Link>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(WingetDisableTestHooks)'=='true'">

```

Comment:
>WingetDisableTestHooks [](http://example.com/codeflow?start=37&length=22)

Is this turned on in our official AppInstaller release build?

Reply:
It will be after this is pulled in.